### ISSUE TITLE
can I load html from String, but css and js from local files in assets?

### ISSUE THREAD

**Issue by eliranwong** [shah-xad/webview_flutter_plus#12]
can I load html from String, but css and js from local files in assets?

I wamt to load html from String .loadString
at the same time, I want to us css and JavaScript local files in assets folder.

is it possible?

is it also possible to bring up a custmised context menu after text is selected?

many thanks for your good work.

**Comment by eliranwong** [comment]
I tried, it doesn't work.

**Comment by shah-xad** [comment]
This package has been `deprecated` in the favor of [webview_flutter](https://pub.dartlang.org/packages/webview_flutter). This package was developed because [webview_flutter](https://pub.dartlang.org/packages/webview_flutter) lacked some important features at that time, like loading local HTML, CSS, and Javascript content from Assets or Strings. But, now [webview_flutter](https://pub.dartlang.org/packages/webview_flutter) provides all this stuff since `webview_flutter: ^2.4.0`. It's highly recommended to use [webview_flutter](https://pub.dartlang.org/packages/webview_flutter) for better performance and better support.

**Comment by shah-xad** [comment]
Hi, I'm sorry but I have to restart this package as I found some issue with webview_flutter. Now, `webview_flutter_plus` is much improved than its previous versions. Please give it a try now. And check if your issue persists is webview_flutter then it'll also be here because it's just an extension of that.

**Comment by ColtonBristow** [comment]
is this fixed ? im not able to render javascript or CSS files. It just says unable to load asset. All assets are included into the yaml file

**Comment by shah-xad** [comment]
Yes, I think, as this plugin uses a local server it should work but I didn't try this. 

`WebViewPlusController` contains a property  `serverPort`, now you can use URL like `http://localhost:<serverPort>`

For example, you have a file index.js in assets, you do something like this

 `http://localhost:<serverPort>/assets/index.js` or   `http://localhost:<serverPort>/packages/your_appname/assets/index.js`.
